From: Gavin Lambert <uecasm@users.noreply.github.com>
Date: Mon, 19 May 2025 02:26:03 +0000
Subject: Add Makefile for dkms package.

---
 dkms/Makefile | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 dkms/Makefile

--- /dev/null
+++ b/dkms/Makefile
@@ -0,0 +1,11 @@
+KERNELRELEASE:=$(shell ls -r /lib/modules/ | head -n1)
+KDIR:=/lib/modules/${KERNELRELEASE}/build
+
+MakeMod = -C ${KDIR} M=${CURDIR}/driver/kernel6.x
+
+all:
+	${MAKE} LVER=${KERNELRELEASE} ${MakeMod} modules
+
+clean:
+	${MAKE} LVER=${KERNELRELEASE} ${MakeMod} clean
+	${MAKE} LVER=${KERNELRELEASE} -C driver/kernel6.x clean_local
